src/domUpdates.js

```diff
@@ -196,90 +208,106 @@ let domUpdates = {
+    else if (event.target.className === "card-apple-icon") { this.addToFavorites(recipeData, user) }
+    else if (event.target.id === "exit-recipe-btn") { this.exitRecipe(fullRecipeInfo) }
+    else if (event.target.id === "instructions") { this.openRecipeInfo(event, fullRecipeInfo, allRecipes, ingredientsData, pantry) }
+  },
+  // openRecipeInfo(event, fullRecipeInfo, allRecipes, ingredientsData, pantry) {
+
+
+  addToCookList(recipeData, user) {
+    let cardId = parseInt(event.target.closest(".recipe-card").id)
+    let card = recipeData.find(recipe => recipe.id === cardId);
+    if (!user.recipesToCook.includes(card)) {
+      event.target.src = "../images/recipeblack.png";
+      user.saveRecipe(card, 'recipesToCook');
+    } else {
+      event.target.src = "../images/recipegreen.png";
+      user.removeRecipe(card, 'recipesToCook');
+    }
+  },
 	
-	addToFavorites(recipeData, user) {
-		let cardId = parseInt(event.target.closest(".recipe-card").id)
-		let card = recipeData.find(recipe => recipe.id === cardId)
-		if (!user.favoriteRecipes.includes(card)) {
-			event.target.src = "../images/apple-logo.png";
-			user.saveRecipe(card, 'favoriteRecipes');
-		} else {
-			event.target.src = "../images/apple-logo-outline.png";
-			user.removeRecipe(card, 'favoriteRecipes');
-		}
-	},
-	//SHOULD THERE BE SOMETHING IN THE WHILE?
-	exitRecipe(fullRecipeInfo) {
-		while (fullRecipeInfo.firstChild && fullRecipeInfo.removeChild(fullRecipeInfo.firstChild)) {
+  addToFavorites(recipeData, user) {
+    let cardId = parseInt(event.target.closest(".recipe-card").id)
+    let card = recipeData.find(recipe => recipe.id === cardId)
+    if (!user.favoriteRecipes.includes(card)) {
+      event.target.src = "../images/apple-logo.png";
+      user.saveRecipe(card, 'favoriteRecipes');
+    } else {
+      event.target.src = "../images/apple-logo-outline.png";
+      user.removeRecipe(card, 'favoriteRecipes');
+    }
+  },
+  //SHOULD THERE BE SOMETHING IN THE WHILE?
+  exitRecipe(fullRecipeInfo) {
+    while (fullRecipeInfo.firstChild && fullRecipeInfo.removeChild(fullRecipeInfo.firstChild)) {
 	
-		}
-		fullRecipeInfo.style.display = "none";
-		document.getElementById("overlay").remove();
-	},
+    }
+    fullRecipeInfo.style.display = "none";
+    document.getElementById("overlay").remove();
+  },
 
-	hideUnsearched(foundRecipes, allRecipes) {
-		this.showAllRecipes(allRecipes);
+  hideUnsearched(foundRecipes, allRecipes) {
+    this.showAllRecipes(allRecipes);
 	  foundRecipes.forEach(recipe => {
-			let domRecipe = document.getElementById(`${recipe.id}`);
+      let domRecipe = document.getElementById(`${recipe.id}`);
 		  domRecipe.style.display = "none";
 	  });
-	},
-
-	toggleMenu(menuOpen) {
-		var menuDropdown = document.querySelector(".drop-menu");
-		menuOpen = !menuOpen;
-		if (menuOpen) {
-			menuDropdown.style.display = "block";
-		} else {
-			menuDropdown.style.display = "none";
-		}
+  },
+
+  toggleMenu(menuOpen) {
```

Comment:
Im wondering if it now doesn't have access to that initial menuOpen = false state and cant recognize when to close it again.  Thanks for pointing that out